Makefile

```diff
@@ -11,6 +11,19 @@ test:
 	@$(MAKE) clean
 	go test ./...
 
+.PHONY: test\:all
+test\:all:
+# Environment variable changes do not invalidate the go test cache, so it is important
+# for us to clean between each run.
+	@$(MAKE) clean
+	CORE_KV_MULTIPLIERS="memory" go test ./...
+	@$(MAKE) clean
+	CORE_KV_MULTIPLIERS="badger" go test ./...
```

Comment:
question: This is equivalent to `none,memory` and `none,badger`?

Reply:
In the old, Defra, terminology, yes.